config/routes.rb

```diff
@@ -14,4 +14,21 @@
     end
   end
   post '/reviews/:review_id/feedback(.:format)' => 'feedback_reviews#create', as: :feedback_reviews
+
+  if (
+      SolidusSupport.api_available? &&
+      Spree::Reviews::Config.draw_api_routes
```

Comment:
Why do you also added the configuration for drawing routes? I don’t think it’s necessary.